src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -0,0 +1,91 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+#pylint: disable=line-too-long
+
+helps['acr'] = """
+            type: group
+            short-summary: Commands to manage Azure container registries.
+            """
+
+helps['acr list'] = """
+            type: command
+            short-summary: List container registries.
```

Comment:
>short-summary: List container registries. [](start = 12, length = 41)

It looks like you rely on the docstring for the parameter help. If that's the case, you can rely on the docstring for the short and long summary. The first sentence will be the short summary. Anything else would be the long summary. (It does look like these entries are better than the existing ones in the docstrings though).